css/apps/media.scss

```diff
@@ -273,4 +273,33 @@
 			}
 		}
 	}
+
+	.file-picker__actions {
+		flex-direction: row-reverse;
+	
+		.upload-picker {
+			display: none;
+		}
+	
+		button {
+			&:disabled {
+				&>.button-vue__wrapper>.button-vue__icon>.material-design-icon.image-plus-icon {
+					background-image: var(--icon-add-dark);
+				}
+			}
+	
+			&>.button-vue__wrapper>.button-vue__icon>.material-design-icon.image-plus-icon {
+				background-image: var(--icon-add-white);
+				width: 44px;
+				height: 44px;
+				background-position: center;
+				background-repeat: no-repeat;
+				background-size: 24px;
+			}
+		}
+	
+		svg {
```

Comment:
we want to affect the icon of only one button, this selector though will hide any svg that appear in .file-picker__actions

Reply:
Fixed